internal/utils/test/mock/realm_client.go

```diff
@@ -61,6 +61,11 @@ type RealmClient struct {
 
 	SchemaModelsFn func(groupID, appID, language string) ([]realm.SchemaModel, error)
 
+	AllowedIPsFn      func(groupID, appID string) ([]realm.AllowedIP, error)
+	CreateAllowedIPFn func(groupID, appID, allowedIP, comment string) (realm.AllowedIP, error)
```

Comment:
If you go with the naming switch in the client, make sure to update these as well.  It would also seem that you still need to implement the mocked functions here